zone/tradeskills.cpp

```diff
@@ -1882,3 +1894,36 @@ bool ZoneDatabase::DisableRecipe(uint32 recipe_id)
 
 	return false;
 }
+
+bool Client::CheckTradeskillLoreConflict(DBTradeskillRecipe_Struct& spec, MySQLRequestResult& results)
+{
+	std::vector< std::pair<uint32,uint8>> lore_conflict_list;
+	std::merge(spec.onsuccess.begin(), spec.onsuccess.end(), spec.onfail.begin(), spec.onfail.end(), std::back_inserter(lore_conflict_list));
+	std::merge(lore_conflict_list.begin(), lore_conflict_list.end(), spec.salvage.begin(), spec.salvage.end(), std::back_inserter(lore_conflict_list));
+
+	for (auto& e : lore_conflict_list) {
+		if (e.first) {
+			auto item_inst = database.GetItem(e.first);
+			if (!item_inst || item_inst->LoreGroup > 0) {
+				continue;
+			}
```

Comment:
Maybe a newline here?

Reply:
Sure, might as well for readability purposes separate these blocks a bit